datafusion/src/physical_plan/file_format/avro.rs

```diff
@@ -54,41 +50,40 @@ pub struct AvroExec {
 
 impl AvroExec {
     /// Create a new JSON reader execution plan provided file list and schema
```

Comment:
```suggestion
    /// Create a new Avro reader execution plan provided file list and schema
```

Reply:
Interesting, I wrote Json in Avro and Avro in Json 😄 Thanks for spotting this!